src/components/basicComponents/BaseButton.vue

```diff
@@ -1,11 +1,14 @@
 <template>
-  <button v-if="!link" :class="mode">
-    <slot></slot>
-  </button>
-  <router-link v-else :to="to" :class="mode">
+  <router-link v-if="link" :to="to" :class="mode">
     <!-- router-link or router-view? -->
     <slot></slot>
   </router-link>
+  <div v-if="likeAButton" :class="mode">
+    <p>{{ text }}</p>
+  </div>
```

Comment:
Ok, I don't get it :D If you need a disabled button you can add a property like that :) This shouldn't be here :)

Reply:
deleted ;)